faceted_search/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "my-app",
   "version": "0.1.0",
   "private": true,
+  "homepage": ".",
```

Comment:
This lets us avoid hardcoding paths in the `.js` itself.  Otherwise, we'd need to also put the path this is served from.  This seemed easy and less headaches for development -- there *were* some tradeoffs involved (https://create-react-app.dev/docs/deployment/#step-1-add-homepage-to-packagejson)

> If you are not using the HTML5 pushState history API or not using client-side routing at all, it is unnecessary to specify the URL from which your app will be served. Instead, you can put this in your package.json: